src/enum_net.cpp

```diff
@@ -178,10 +178,9 @@ namespace {
 	{
 		if (family == AF_INET)
 		{
-			using bytes_t = boost::asio::ip::address_v4::bytes_type;
-			bytes_t b;
-			std::memset(&b[0], 0xff, b.size());
-			for (int i = int(b.size()) - 1; i > 0; --i)
+			address_v4::bytes_type b;
+			b.fill(0xff);
+			for (std::size_t i = b.size() - 1; i > 0; --i)
```

Comment:
hi @ssiloti, I want to confirm that `i > 0` (with `>`) is the correct condition, thanks

Reply:
No, it's not. The loop needs to run when i = 0 to set the least significant byte's value correctly if `bits` is less than 8 bits smaller than the size of the address.

Reply:
I will change that then